google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -212,15 +212,16 @@ public static void beforeClass() throws IOException {
                         LifecycleCondition.newBuilder().setAge(1).build())))
             .build());
 
+    storage.create(BucketInfo.newBuilder(BUCKET_REQUESTER_PAYS).build());
+
     // Prepare KMS KeyRing for CMEK tests
     prepareKmsKeys();
   }
 
-  @Before
-  public void beforeEach() {
+  private static void unsetRequesterPays() {
```

Comment:
Why not leave as-is with `Before` annotation? Otherwise it can be easily missed when new requester pays tests are added.

Reply:
@frankyn 
The purpose of `@Before` annotation here is to invoke the method to mend the shared `BUCKET` after requester-pays tests before *each* test. With this cleanup fix, all the requester-pays tests are "moved" to their dedicated folder, it's no longer required to mend the `BUCKET`.

Actually, it's not absolutely necessary to invoke `unsetRequesterPays` before requester-pays tests, these tests will work without it. Invocation is only needed to make `setRequesterPays(true)` change the value from `not true` to `true`. So it's not critical if a new requester-pays test misses calling `unsetRequesterPays()`.